extensions/amp-story/1.0/amp-story-system-layer.js

```diff
@@ -340,10 +390,38 @@ export class SystemLayer {
    * @private
    */
   onMutedStateUpdate_(isMuted) {
+    
     this.vsync_.mutate(() => {
-      isMuted ?
-        this.getShadowRoot().setAttribute(AUDIO_MUTED_ATTRIBUTE, '') :
-        this.getShadowRoot().removeAttribute(AUDIO_MUTED_ATTRIBUTE);
+      if(isMuted) {
+        this.getShadowRoot().setAttribute(AUDIO_MUTED_ATTRIBUTE, 'muted');
+       } else if(this.storeService_.get(StateProperty.PAGE_HAS_AUDIO_STATE)) {
+        this.getShadowRoot().setAttribute(AUDIO_MUTED_ATTRIBUTE, 'soundon');
```

Comment:
Is this (and `nosound` below) only used for testing?

Reply:
Sorry if I'm misinterpreting this comment, but the reason I have them is so for the css file to know which of the three messages to show for muted. I need them to know the audio state of whether it is muted, unmuted without sound, or unmuted with sound instead of just the binary mute/unmute from before. Sorry if this was wrong on confusing :)

Reply:
Sounds good!  Sorry about the confusion; I hadn't seen the CSS changes because GitHub deemed them too large and I didn't realize

Reply:
no worries :)